### DISCUSSION TITLE
Image Slider taking too long when changing between images

### DISCUSSION THREAD

**Discussion by henriquechagasg** [vercel/next.js#24526] [Help]
I used the Image component provided for next/image to make an image slider component, but when I click to switch between images, it's taking too long until the image loads . Is there any way to fix that?

**Comment by bobarros** [comment]
How large are they?

Did you try to change loading from lazy to eager?

Check here --> https://nextjs.org/docs/api-reference/next/image

It's going to explain how to change loading from lazy to eager in your Image Component.
